src/components/ViewPCOL/NotesForm.tsx

```diff
@@ -28,30 +26,28 @@ const NotesForm = ({
   isOpen: boolean;
   closeDrawer: () => void;
 }) => {
-  // const { program, pcolId } = useParams();
-  // const pcol = usePCOL(String(program), Number(pcolId));
-  // const addNote = useAddNote(pcolId);
-  // const sendMail = useSendEmail();
+  const { program, pcolId } = useParams();
+  const addNote = useAddNote(String(program), Number(pcolId));
+  const sendMail = useSendEmail();
 
-  // const [toSelections, setToSelections] = useState<Person[]>([]);
-  // const [ccSelections, setCcSelections] = useState<Person[]>([]);
+  const [toSelections, setToSelections] = useState<Person[]>([]);
+  const [ccSelections, setCcSelections] = useState<Person[]>([]);
   const [newNoteText, setNewNoteText] = useState("");
 
-  // const subject =
-  //   "Action Required for PCOL " + pcolId;
+  const subject = "PCOL note added: " + pcolId;
```

Comment:
This creates a subject line such as "PCOL note added: 12" which would not have much meaning to the user, as the pcolId is just a fully internal number that isn't part of the PCOL identifier they would normally reference. Should be more like:
"FA8770-2025136-1"